<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-P2QE43U4" failures="1" tests="3" name="Test11" time="54.864" errors="0" timestamp="2024-10-22T14:01:21 CDT">
  <testcase classname="TestClasses.Registrationrerun" name="Registrationpage" time="4.728">
    <skipped/>
  </testcase> <!-- Registrationpage -->
  <testcase classname="TestClasses.Registrationrerun" name="Registrationpage" time="1.844">
    <skipped/>
  </testcase> <!-- Registrationpage -->
  <testcase classname="TestClasses.Registrationrerun" name="Registrationpage" time="3.157">
    <failure type="java.lang.AssertionError" message="expected [Credentials already exists] but found [password_mismatch:The two password fields didn’t match.]">
      <![CDATA[java.lang.AssertionError: expected [Credentials already exists] but found [password_mismatch:The two password fields didn’t match.]
at TestClasses.Registrationrerun.Registrationpage(Registrationrerun.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- Registrationpage -->
</testsuite> <!-- Test11 -->
